modules/i3s/src/lib/parsers/parse-i3s.js

```diff
@@ -31,9 +31,20 @@ export function normalizeTileData(tile, options, context) {
 
 export function normalizeTileNonUrlData(tile) {
   scratchCenter.copy(tile.mbs);
-  const centerCartesian = Ellipsoid.WGS84.cartographicToCartesian(tile.mbs.slice(0, 3));
+  const sphere = [
+    ...Ellipsoid.WGS84.cartographicToCartesian(tile.mbs.slice(0, 3)), // cartesian center of sphere
+    tile.mbs[3] // radius of sphere
+  ];
+  const box = tile.obb
+    ? [
+        ...Ellipsoid.WGS84.cartographicToCartesian(tile.obb.center), // cartesian center of box
+        ...tile.obb.halfSize, // halfSize
+        ...tile.obb.quaternion // quaternion
+      ]
+    : undefined;
   tile.boundingVolume = {
```

Comment:
Is this being converted in a "normalized" tile format? If so it should be documented in https://loaders.gl/modules/tiles/docs. 

This can be done in a separate PR.

@belom88 may be able to provide you with more context about what I mean.

Reply:
Yes, I think the idea was that while the OBB is always internally stored in one of these representations,  the OBB can be constructed from either representation (using the `from...` methods) and there are accessors that lets you get back either view (with small performance penalty if conversion of course).

Does that help?